Buildings/Experimental/DHC/EnergyTransferStations/Cooling/DirectUncontrolled.mo

```diff
@@ -0,0 +1,175 @@
+within Buildings.Experimental.DHC.EnergyTransferStations.Cooling;
+model DirectUncontrolled "Direct cooling ETS model for district energy systems 
+  without in-building pumping or deltaT control"
+  extends
+    Buildings.Experimental.DHC.EnergyTransferStations.BaseClasses.PartialETS(
+    final typ=DHC.Types.DistrictSystemType.Cooling,
+    final have_weaBus=false,
+    final have_chiWat=true,
+    final have_heaWat=false,
+    final have_hotWat=false,
+    final have_eleHea=false,
+    final nFue=0,
+    final have_eleCoo=false,
+    final have_pum=false,
+    final have_fan=false,
+    nPorts_aChiWat=1,
+    nPorts_bChiWat=1);
+  parameter Modelica.Units.SI.MassFlowRate m_flow_nominal(
+    final min=0,
+    final start=0.5)
+    "Nominal mass flow rate"
+    annotation(Dialog(group="Nominal condition"));
+  // Pressure drops
+  parameter Modelica.Units.SI.PressureDifference dpSup(
+    final min=0,
+    displayUnit="Pa")=5000
+    "Pressure drop in the ETS supply side";
+  parameter Modelica.Units.SI.PressureDifference dpRet(
+    final min=0,
+    displayUnit="Pa")=5000
+    "Pressure drop in the ETS return side";
+  Modelica.Blocks.Interfaces.RealOutput Q_flow(
```

Comment:
See the comments in `DirectController` model.

Reply:
Thanks, I have moved the icons to avoid the overlap problem.